sources/packages/web/src/views/student/StudentProfile.vue

```diff
@@ -87,6 +111,10 @@ export default {
       () => studentAllInfo.value.pdStatus === StudentPDStatus.Pending,
     );
 
+    const saveLabel = computed(() =>
+      props.editMode ? "Save profile" : "Create profile",
+    );
+
     const getStudentDetails = async () => {
       if (props.editMode) {
```

Comment:
Like what we saw in offerings, create and edit can be handled by different vue components here. But that is not part of the story.

Reply:
Not part of this PR but will be done alongside with BCeID effort because the same form will be reused for Ministry approval and also for the future student edit by the Ministry.